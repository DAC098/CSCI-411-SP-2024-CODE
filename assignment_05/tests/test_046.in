40 692
101
403
741
909
1253
1284
1383
1450
2024
2350
2399
2552
2687
2816
2967
3129
3771
3924
4040
4121
4744
5012
5225
5584
6219
6263
6396
6711
7113
7142
7228
7415
7876
8319
8460
8615
8850
8904
9227
9376
