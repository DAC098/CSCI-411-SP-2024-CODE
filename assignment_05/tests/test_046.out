Enter the number of hotels and the ideal number of miles to travel per day: Enter 40 hotel distances each on a separate line: Hotels to visit: 3 7 9 12 16 18 21 24 27 30 33 36 40 
Total penalty: 122570
